vars/buildYumRepo.groovy

```diff
@@ -0,0 +1,27 @@
+void call(Map args = [:]) {
+    lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
+
+    def buildManifest = lib.jenkins.BuildManifest.new(readYaml(file: args.buildManifest))
+    echo "Creating repo file and data for ${args.buildManifest}"
```

Comment:
`Create yum repo metadata and repo file for ......`

Reply:
updated